jwst/extract_1d/extract.py

```diff
@@ -134,13 +134,12 @@ def open_extract1d_ref(refname: str, exptype: str) -> dict:
 
     if refname == "N/A":
         ref_dict = None
-    elif exptype in ['MIR_MRS', 'NRS_IFU']:
+    elif exptype in ['MIR_MRS', 'NRS_IFU', 'NRS_LAMP']:
```

Comment:
I'm not sure this simple addition will work for all modes. It will be OK for lamp exposures that happen to be taken in IFU mode (OPMODE='IFU'), but what about lamp exposures taken in MOS or FS mode (OPMODE='MSASPEC' or 'FIXEDSLIT'). We don't want those latter types going down the branch that only works for IFU mode.